src/cluster.cpp

```diff
@@ -224,20 +219,6 @@ namespace zim
           break;
         }
 
-      case zimcompBzip2:
-        {
-#if defined(ENABLE_BZIP2)
-          log_debug("uncompress data (bzip2)");
-          zim::Bunzip2Stream is(in);
-          is.exceptions(std::ios::failbit | std::ios::badbit);
-          read_header(is);
-          read_content(is);
-#else
-          throw std::runtime_error("bzip2 not enabled in this library");
```

Comment:
I would have keep this to properly handle a potential zim file with bzip2 compression.
Maybe changing a bit the error message and moving the `case` at the end of the `switch` but while keeping the same behavior (throw a std::runtime_error)